oc-cluster

```diff
@@ -253,7 +303,15 @@ function up {
 
     echo "$CMDLINE" > $OPENSHIFT_PROFILES_DIR/$_profile/run
     echo "$(<$OPENSHIFT_PROFILES_DIR/$_profile/run)"
-    $CMDLINE
+    if [ ! -z $DEV_DOCKER_HOST ]; then
+      echo "Running in developer mode, using DOCKER_HOST=$DEV_DOCKER_HOST"
+      if [[ "$__PLATFORM" == "linux" ]]; then
+        CMDLINE="DOCKER_HOST=$DEV_DOCKER_HOST $CMDLINE"
+      fi
+      $CMDLINE -e DOCKER_HOST=$DEV_DOCKER_HOST
```

Comment:
The thing is that it would require an additional parameter or way of configuring. I would like a way to have a single template that would work for everybody. In any case I would maybe modify the bindmountproxy to detect if the src of a bind is empty not do a bind. Then it's just a matter of having empty values for the envs that could replace the tokens or else have just the template example showing what can be done and the user have the template they want. 
Happy to understand what options you see.